docs/gettingstarted/quickstart.ipynb

```diff
@@ -89,23 +89,35 @@
     "!pip install aiohttp --quiet"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "611cd2f6",
+   "metadata": {},
+   "source": [
+    "For the purpose of this tutorial we will only read a small 1 degree cone region of Gaia DR3, one that should contain our objects. While LSDB typically loads only the minimal amount of data it needs for our workflow, manually providing it with spatial information helps it identify which files to search for on disk."
```

Comment:
loads -> loads into memory, right?. It is a bit unclear, we use verbs like load and read in a bit of a unclear fashion (all of us).

Reply:
You're right, I unified the terms and clarified this sentence!